src/config.py

```diff
@@ -155,6 +164,22 @@ def __init__(self, print_warnings: bool = False):
             "melee_min_score": int(self._select_val("claws", "melee_min_score")),
         }
 
+    def parse_item_config_string(self, key: str = None) -> ItemProps:
+        item_props = ItemProps()
+        # split string by commas NOT contained within parentheses
+        item_string_as_list = re.split(r',\s*(?![^()]*\))', self._select_val("items", key).upper())
+        trim_strs=["AND(", "OR(", "(", ")", " "]
+        clean_string = [re.sub(r'|'.join(map(re.escape, trim_strs)), '', x).strip() for x in item_string_as_list]
+        item_props.pickit_type = int(clean_string[0])
+        try:
+            item_props.include = clean_string[1].split(',') if clean_string[1] else None
+            item_props.include_type = "AND" if "AND" in item_string_as_list[1] else "OR"
+        except: pass
+        try:
+            item_props.exclude = clean_string[2].split(',') if clean_string[2] else None
+            item_props.exclude_type = "AND" if "AND" in item_string_as_list[2] else "OR"
+        except: pass
```

Comment:
I dont see when these would fail, why the try except? It's also a bit risky to hide unwanted errors. If we are allow a specific error, e.g. ValueError, specify that one. Right now if there is an issue with these statements we will never see it.

Reply:
Maybe even add print the error at least to the debug log

Reply:
this is to handle pickit.ini lines that have variable # of fields.
so I can say `=1` `=1, ethereal` and `=1, socketed, ethereal`

a little lazy way to go about it I guess =/

Reply:
Okay, then do except IndexError. This way you will still see errors that are not expected.